### ISSUE TITLE
Budget identifier

### ISSUE THREAD

**Issue by markbrough** [pwyf/2016-index-data-quality-tests#35]
**Description**
The budget classification is a way of linking the activity to the recipient country government’s own budget codes. This is only expected if the activity is at least in the implementation phase, and if the aid type is not budget support.

**Existing tests**

```
at least one country-budget-items[@vocabulary="1"]/budget-item/@code is on list BudgetIdentifier (if activity-status/@code is at least 2 and (default-aid-type/@code or transaction/aid-type/@code is not A01 or A02))?
capital-spend exists (if activity-status/@code is at least 2 and (default-aid-type/@code or transaction/aid-type/@code is not A01 or A02))?
```

**Issues**
1. There have been changes in the budget identifier methodology since the 2014 Index.
2. The principal change (for the purposes of the Index) is to fold the Common Code codes into the CRS sector code list, for those codes that are more disaggregated than the existing CRS sector codes.

**2016 Aid Transparency Index test**
We are not planning to change this test in 2016, given the timing, and the fact that WP-STAT is yet to decide on the proposal. We could perhaps later consider whether the CRS sector codes used are sufficiently detailed to map to country budget classifications.

**Comment by rachelrank** [comment]
I agree it makes sense to hold off until the proposal to fold the Common Codes into the CRS sector list is formally approved by WP-STAT; but let's keep this as a placeholder in future years as this work progresses and based on feedback from users.

**Comment by YohannaLoucheur** [comment]
Not changing the test means that the presence of data in the Budget ID field ("country-budget-items") would be checked along with Capital Expenditure ("capital-spend"), is that correct?  

Given the proposal presented to WP-STAT, which would lead to the country-budget-item field not longer being used, I believe it would be better to remove the country-budget-items part for 2016 and only test capital-spend. Publishers are unable to use the new codes until the revised codelist is available, so it is not fair to assess their use (especially as it counts for 3.25% of the total score). On the other hand, nothing stops publishers from using the capital-spend field.

**Comment by jamieattard** [comment]
We did attempt to map our OECD CRS coding to the Budget ID, but as can be appreciated the mapping was not complete because of the absence of adequate CRS codes.  We don't presently report capital expenditure as we do not capture this systematically.

**Comment by rjboles** [comment]
Until the budget identifier is formally agreed, we are not in a position to begin reporting on it and therefore, it would be better to remove this element from the index

**Comment by YohannaLoucheur** [comment]
Just to be clear, the "capital-spend" field - which is half of the "Budget Identifier" - is agreed and can already be used. There is no reason for publishers not to start collecting and publishing this data, even partially (ie even if not all projects are coded).  
Only the "country-budget-item" part of the Budget Identifier remains to be confirmed.

**Comment by TreasuryOTA** [comment]
This does not apply to us. Are we scoring zero points for this category?

**Comment by markbrough** [comment]
Thanks for all of the comments.

**For the 2016 Aid Transparency Index**

We will drop the first test, to check for `country-budget-items`. We will retain the second test in its existing form, to check for `capital-spend`

_Notes_
- We agree with @YohannaLoucheur that, considering the adjustments to the Budget Identifier methodology over the last year, it does not make sense to still test for the `country-budget-items` element.
- Given that WP-STAT has not yet approved the proposed changes to purpose codes, it would not yet be reasonable to test for the usage or otherwise of the proposed codes.
- Given that the IATI Steering Committee has already approved the use of the `capital-spend` element in the IATI Standard, and that there are no changes proposed to this element, it is reasonable to continue to test for this element.
